pkg/cmd/pr/view/view.go

```diff
@@ -223,6 +232,30 @@ func printHumanPrPreview(opts *ViewOptions, pr *api.PullRequest) error {
 		fmt.Fprintln(out, pr.Milestone.Title)
 	}
 
+	// Auto-Merge status
+	autoMerge := pr.AutoMergeRequest
+	if autoMerge != nil {
+		var mergeMethod string
+		switch autoMerge.MergeMethod {
+		case "MERGE":
+			mergeMethod = "creating a merge commit"
+		case "REBASE":
+			mergeMethod = "rebase and merge"
+		case "SQUASH":
+			mergeMethod = "squash and merge"
+		default:
+			mergeMethod = fmt.Sprintf("an unknown merge method (%s)", autoMerge.MergeMethod)
+		}
+		fmt.Fprintf(out,
+			"%s %s by %s, using %s • %s\n",
+			cs.Bold("Auto-merge:"),
+			cs.Green("enabled"),
+			autoMerge.EnabledBy.Login,
+			mergeMethod,
+			text.FuzzyAgo(opts.Now(), autoMerge.EnabledAt),
```

Comment:
How important do we think showing the date the auto-merge was enabled at is? Feels a bit insignificant to me.

Reply:
Agreed. I'll drop that part. I was simply riffing of the other lines without sufficient thought as to what it would add.